app/mixins/inplace2.js

```diff
@@ -0,0 +1,58 @@
+import Ember from 'ember';
```

Comment:
I'm ok with splitting these up and removing the mixin entirely or partially. This might be a place where repeating yourself is a good thing.